node-cli/index.js

```diff
@@ -0,0 +1,70 @@
+const Realm = require("realm");
+const inquirer = require("inquirer");
+const users = require("./users");
+const schemas = require("./schemas");
+const output = require("./output");
+
+/*  Change the logLevel to increase or decrease the 
+    amount of messages you see in the console.
+    Valid options are:
+    fatal, error, warn, info, detail, debug, and trace
+*/
+Realm.Sync.setLogLevel("error");
```

Comment:
You should set this *after* creating the instance of `Realm.App`. We need to update the documentation (we have a ticket).